authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.adoc

```diff
@@ -86,11 +86,6 @@ data:
 <3> The resource URL of the service account that can be impersonated with these credentials.
 <4> The path to the service account token inside the pod. By convention, this is `/var/run/secrets/openshift/serviceaccount/token` for {product-title} components.
 
-[IMPORTANT]
-====
-In {product-title} 4.10.8, image registry support for using GCP Workload Identity was removed due to the discovery of link:https://bugzilla.redhat.com/show_bug.cgi?id=2069807[an adverse impact to the image registry]. To use the image registry on an {product-title} 4.10.8 cluster that uses Workload Identity, you must configure the image registry to use long-lived credentials instead. This mitigation is planned to last until Workload Identity support for the image registry is restored in a later release.
```

Comment:
Do we want to remove this or call out what versions it wasn't supported in?

Reply:
Hi @sferich888,

I removed it from the documentation and updated the verbiage in the release notes. I figured since it is updated in the release notes, I could remove this paragraph from the documentation. Should I change it to reflect the changes in the release notes?

Reply:
I think we should be conveying when this was and wasn't supported, that way there is no confusing about when/where we they can use this functionality.

Reply:
@sferich888 Done! :)